app/services/prompt_augmentor.rb

```diff
@@ -10,14 +10,26 @@ def augment
     link_message_with_augmentations
   end
 
-  def prompt
+  def prompt # rubocop:todo Metrics
```

Comment:
Can you address the Rubocop exemptions? There are many in the codebase now, but I'm trying hard not to add more.

There are four in this file and this one is pretty broad, and I worry the code will end up littered with them if we're not more disciplined.

Reply:
Addressed rubocop in app code, but rspec was driving me a little crazy since I couldn't set `relevant` in the search hit object. I tried `Search::SearchHit.new(graph_entity, 240.0, relevant: true)` but that didn't work.

I added a check to avoid updating the message prompt with `Query: ...` when there are no relevant hits, which was confusing.  
Maybe it should add `The query found hits, but none were relevant` in that case?